### DISCUSSION TITLE
Status Map for Non Time Series Data

### DISCUSSION THREAD

**Discussion by ShilpaSKavasseri** [flant/grafana-statusmap#228] [Q&A]
I am trying to plot a status map with **non series data**. X Axis and Y Axis labels would be discrete categories and not time series data. The data source used here is MYSQL. MYSQL query with non time series data are not getting displayed and throws an error. It would be of great help  if you can help me sort this out. Thanks in advance

@diafour @gsmetal

**Comment by diafour** [comment]
Hello! Statusmap cannot display such data, it expects timestamps for X axis to show statuses over time, not an instant state. Take a look at the [boomtable](https://grafana.com/grafana/plugins/yesoreyeram-boomtable-panel/) plugin.
